### PR TITLE
[WIP] Announce Configuration pages migration to Bootstrap

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit f485599f398011d6ad6c5edff68f91ab830bddad

https://deploy-preview-219--openbuildservice.netlify.com

File: _posts/development/2019-03-03-revamping-configuration-ui.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# And Much More
+
+We addressed some issues we identified and gathered from your feedback. Here’s a list of what has been done.
+
+When accessing to the linking projects an error was raised because that part of the view wasn't migrated to Bootstrap. Now it is and it looks like this:
+
+<img src="/images/posts/revamping-configuration-ui/linking_projects.png" alt="Linking projects modal" style="width: 80%;" class="center" />
+
+In the pagackages' table, some dates should be formatted using words, for example _"7 minutes ago"_. However, it was only happing on the first page of the table. Now, it is properly formatted on every page.
```
Comment (bgeuken) [inline _posts/development/2019-03-03-revamping-configuration-ui.md]: typo: pa**ga**ckages

**Comment by saraycp** [comment]
[Communicate the deliverable to the customers in a proper way](https://trello.com/c/7q8gWbEX/1060-communicate-the-deliverable-to-the-customers-in-a-proper-way)

File: _posts/development/2019-03-03-revamping-configuration-ui.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Moving Forward in the User Interface Revamp! Now it is Configuration Pages Turn
+category: development
+---
+
+{% include partials/_revamping-ui-releases-history.md index="4" %}
+
+We have given a step forward in order to revamp the whole OBS interface! :tada:
```
Comment (bgeuken) [inline _posts/development/2019-03-03-revamping-configuration-ui.md]: *We have taken

File: _posts/development/2019-03-03-revamping-configuration-ui.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# And Much More
+
+We addressed some issues we identified and gathered from your feedback. Here’s a list of what has been done.
+
+When accessing to the linking projects an error was raised because that part of the view wasn't migrated to Bootstrap. Now it is and it looks like this:
+
+<img src="/images/posts/revamping-configuration-ui/linking_projects.png" alt="Linking projects modal" style="width: 80%;" class="center" />
+
+In the pagackages' table, some dates should be formatted using words, for example _"7 minutes ago"_. However, it was only happing on the first page of the table. Now, it is properly formatted on every page.
```
Comment (dmarcoux) [inline _posts/development/2019-03-03-revamping-configuration-ui.md]: happing -> happening

**Comment by dmarcoux** [comment]
I will take over since @saraycp is on holiday.

**Comment by dmarcoux** [comment]
I've just pushed changes. The only missing part is the screenshot for the `Interconnect` section. Everything else is done and ready to be reviewed.

I have omitted the notifications page since it is almost an exact copy of the user notifications which we presented in a previous blog post. The announcements were also skipped since they are pretty basic. I don't believe it's interesting to our users. About the issues, I listed the ones which should interest our users the most. Weird, trivial or really specific issues were skipped, so things like the bug with `nextstatus` param, changing the title of an icon or fixing the multiple popovers are such examples.

Let me know what you think.

**Comment by bgeuken** [comment]
Thanks. I'll add a short section regarding the datatable changes.

**Comment by eduardoj** [comment]
> @openSUSE/obs-frontend, let me know what you think.

Fine for me. Thanks! Waiting for the screenshot for the `Interconnect` section.

**Comment by dmarcoux** [comment]
I added the missing screenshot for the `Interconnect` section.

**Comment by dmarcoux** [comment]
I just realized that we will never be able to merge this PR since @saraycp is on holiday because it's a draft PR and she's the only one who can change the state of the PR. I have to create another PR.

**Comment by dmarcoux** [comment]
Trying if I can change the state after reopening... nope....

**Pull request closed without merging**